client/src/components/LikeButton.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import { PiHeartFill } from 'react-icons/pi';
+import { styled } from 'styled-components';
+import client, { ClientNetworkError } from '../client';
+import { Cafe } from '../types';
+
+type LikeProps = {
+  currentLikeCount: number;
+};
+
+const LikeButton = ({ currentLikeCount }: LikeProps) => {
```

Comment:
```suggestion
type LikeButtonProps = {
  currentLikeCount: number;
};

const LikeButton = ({ currentLikeCount }: LikeButtonProps) => {
```
`LikeButton` + `Props` 라는 식으로 네이밍하면 더욱 예측 가능하고 좋을 것 같습니다!

Reply:
아 원래 컴포넌트 이름이 Like 였다가 중간에 LikeButton으로 변경했는데요...! 그래서 누락이 있었던 거 같습니당 ^^; 수정하도록 하겠습니닷

Reply:
굿~ 좋아용 🫡